taxi/views.py

```diff
@@ -87,3 +89,54 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+    def get_success_url(self) -> str:
```

Comment:
it is not necessary to set `get_success_url` if `get_absolute_url` is defined in your model.